File: java/yapay-gateway-client/src/main/java/br/com/yapay/gateway/restv3/model/AddressData.java
package br.com.yapay.gateway.restv3.model;

import com.google.gson.annotations.SerializedName;

public class AddressData {

	@SerializedName("logradouro")
	private String street;

	@SerializedName("numbero")
	private String number;

	@SerializedName("complemento")
	private String complement;

	@SerializedName("cep")
	private String zipCode;

	@SerializedName("bairro")
	private String district;

	@SerializedName("cidade")
	private String city;

	@SerializedName("estado")
	private String state;

	@SerializedName("pais")
	private String country;

	public String getStreet() {
		return street;
	}

	public void setStreet(String street) {
		this.street = street;
	}

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}

	public String getComplement() {
		return complement;
	}

	public void setComplement(String complement) {
		this.complement = complement;
	}

	public String getZipCode() {
		return zipCode;
	}

	public void setZipCode(String zipCode) {
		this.zipCode = zipCode;
	}

	public String getDistrict() {
		return district;
	}

	public void setDistrict(String district) {
		this.district = district;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public String getState() {
		return state;
	}

	public void setState(String state) {
		this.state = state;
	}

	public String getCountry() {
		return country;
	}

	public void setCountry(String country) {
		this.country = country;
	}

}

File: net/YapayGatewayLib/model/AddressData.cs
using System;
using Newtonsoft.Json;

namespace YapayGatewayLib.Model
{
    public class AddressData
    {
        [JsonProperty(PropertyName = "logradouro")]
        public string street { get; set; }

        [JsonProperty(PropertyName = "numero")]
        public string number { get; set; }

        [JsonProperty(PropertyName = "complemento")]
        public string complement { get; set; }

        [JsonProperty(PropertyName = "cep")]
        public string zipCode { get; set; }

        [JsonProperty(PropertyName = "bairro")]
        public string district { get; set; }

        [JsonProperty(PropertyName = "cidade")]
        public string city { get; set; }

        [JsonProperty(PropertyName = "estado")]
        public string state { get; set; }

        [JsonProperty(PropertyName = "pais")]
        public string country { get; set; }
    }
}
